aspnetcore/mvc/views/tag-helpers/intro.md

```diff
@@ -126,6 +126,10 @@ In the code image below, the Tag Helper prefix is set to `th:`, so only those el
 
 The same hierarchy rules that apply to `@addTagHelper` also apply to `@tagHelperPrefix`.
 
+## Self-closing Tag Helpers
+
+In general, Tag Helpers can't be used as self-closing tags. Self-closing a Tag Helper results in a self-closing tag in the rendered output. For more information, see [this note](xref:mvc/views/tag-helpers/authoring#self-closing) in [Authoring Tag Helpers](xref:mvc/views/tag-helpers/authoring).
```

Comment:
I think it's more appropriate to say that you should use care when using a tag helper as a self-closing tag. Some tag helpers are designed to be self-closing tags, others are not, and it will suppress their output.